packages/forklift-console-plugin/src/modules/Overview/views/overview/tabs/Metrics/cards/ChartsCard.tsx

```diff
@@ -27,26 +27,58 @@ interface MigrationDataPoint {
   value: number;
 }
 
+enum TimeRangeOptions {
+  Last7Days = 'Last7Days',
+  Last31Days = 'Last31Days',
+  Last24H = 'Last24H',
+}
+
 const toStarted = (m: V1beta1Migration): string => m.status.started;
 const toFinished = (m: V1beta1Migration): string => m.status.completed;
 const hasTimestamp = (timestamp: string) => timestamp && DateTime.fromISO(timestamp).isValid;
 const toDateTime = (timestamp: string): DateTime => DateTime.fromISO(timestamp);
 const isLast7Days = (date: DateTime) => date.diffNow('days').get('days') <= 7;
+const isLast31Days = (date: DateTime) => date.diffNow('days').get('days') <= 31;
 const isLast24H = (date: DateTime) => date.diffNow('hours').get('hours') <= 24;
 const toDayLabel = (date: DateTime): string => date.toFormat('LLL dd');
 const toHourLabel = (date: DateTime): string => date.toFormat('HH:mm');
-const createTimeBuckets = (isDaysViewSelected: boolean) =>
+
+const TimeRangeOptionsDictionary = {
+  Last7Days: {
+    label: 'Migrations (last 7 days)',
+    span: { days: 7 },
+    bucket: { day: 1 },
+    unit: 'day',
+    filter: isLast7Days,
+  },
+  Last31Days: {
+    label: 'Migrations (last 31 days)',
+    span: { days: 31 },
+    bucket: { day: 4 },
+    unit: 'day',
+    filter: isLast31Days,
+  },
+  Last24H: {
+    label: 'Migrations (last 24 hours)',
```

Comment:
```suggestion
     // t ('Migrations (last 24 hours)')
    labelKey: 'Migrations (last 24 hours)',
```
if we want to translate it, we can add a `t` comment, and rename to `key` 
a `t` comment will add the string to the dictionary when running `npm run i18n`